hardhat.config.ts

```diff
@@ -47,18 +47,40 @@ const config: HardhatUserConfig = {
         version: "0.8.17",
         settings: {
           optimizer: { enabled: true, runs: 800 },
+          viaIR: true,
         },
       },
     ],
   },
   networks: {
     hardhat: {
+      // Normal Config
       accounts: {
         accountsBalance: "10000000000000000000000000",
         //   mnemonic: MNEMONIC,
       },
       allowUnlimitedContractSize: true,
       chainId: 31337,
+
+      // Forking Config for Deployment Testing
+      // chainId: 84531,
+      // forking: {
+      //   url: "https://base-goerli.blockpi.network/v1/rpc/public",
+      // },
+      // accounts: [
+      //   {
+      //     privateKey: process.env.PRIVATE_KEY!,
+      //     // This is a dummy value and will be overriden in the test by
+      //     // the account's actual balance from the forked chain
+      //     balance: "10000000000000000000000000",
+      //   },
+      // ],
```

Comment:
Uncertain about keeping that commented code

Reply:
I agree, this is not ideal. I'll think of a good way to programmatically switch b/w normal mode and simulation mode